context/response/responses.go

```diff
@@ -0,0 +1,53 @@
+package response
+
+import (
+	beecontext "github.com/astaxie/beego/context"
+)
+
+// JSON renders value to the response as JSON
+func JSON(value interface{}, encoding ...bool) Renderer {
```

Comment:
> c.Ctx.Output.Redirect feels to verbose IMO

When I design the Ctx, we just name request as input and response as output. we need to keep consistency with the design. 

> Also, response.JSON can not be put on Output because Output already has JSON. and repsonse JSON again returns a renderer which eventually calls Output.JSON.

If you return the render from the JSON, I would like to rename it to `JSONRenderer`